esp32c3-hal/examples/spi_loopback_dma.rs

```diff
@@ -106,10 +106,11 @@ fn main() -> ! {
 
         let transfer = spi.dma_transfer(send, receive).unwrap();
         // here we could do something else while DMA transfer is in progress
-        let mut i = 0;
+        let mut n = 0;
```

Comment:
IMHO we should rename this variable in this example for all other chips, too if we do it here

Reply:
Agreed - that was the plan when I copied the new example to the other chips' directories. In this function, i is being used for three different things now - this change at least drops that to 2.